dll/win32/comctl32/static.c

```diff
@@ -626,7 +626,15 @@ static void STATIC_PaintTextfn( HWND hwnd, HDC hdc, DWORD style )
     }
     else
     {
-        DrawTextW( hdc, text, -1, &rc, format );
+        UINT flags = DST_TEXT | ((style & WS_DISABLED) ? DSS_DISABLED : DSS_NORMAL);
```

Comment:
```suggestion
        UINT flags = /*DST_TEXT |*/ ((style & WS_DISABLED) ? DSS_DISABLED : DSS_NORMAL);
```
So, yes, maybe this `DST_TEXT` flag is not necessary, I suggest you comment it out temporarily for testing.